apache/metron
# METRON-1253: Manual pasting of timestamps into the timestamp picker

## Contributor Comments

Link to the original ASF Jira ticket: https://issues.apache.org/jira/browse/METRON-1253

Currently, it's not possible to write or paste a valid date to a date selector powered input field. This PR has been intended to solve this inconvenience. Please make sure you add a valid date format which is `YYYY-MM-DD HH:mm:ss`

Testing locally
==========

1. Start the Metron Alerts UI
2. Go to `http://localhost:4201/alerts-list` in the browser
3. Open the date range selector floating pane at the upper right
4. In the floating window, try to manually write or paste valid date formats within the accepted time range (In the two input fields on the left side of the floating window)
5. It should be nicely done. 🙌


## Pull Request Checklist

Thank you for submitting a contribution to Apache Metron.  
Please refer to our [Development Guidelines](https://cwiki.apache.org/confluence/pages/viewpage.action?pageId=61332235) for the complete guide to follow for contributions.  
Please refer also to our [Build Verification Guidelines](https://cwiki.apache.org/confluence/display/METRON/Verifying+Builds?show-miniview) for complete smoke testing guides.  


In order to streamline the review of the contribution we ask you follow these guidelines and ask you to double check the following:

### For all changes:


### For code changes:
  ```
  mvn -q clean integration-test install && dev-utilities/build-utils/verify_licenses.sh 
  ```


### For documentation related changes:

  ```
  cd site-book
  mvn site
  ```

#### Note:
Please ensure that once the PR is submitted, you check travis-ci for build issues and submit an update to your PR as soon as possible.
It is also recommended that [travis-ci](https://travis-ci.org) is set up for your personal repository such that your branches are built there before submitting a pull request.
